src/index.js

```diff
@@ -182,6 +183,12 @@ function toJavaScript(psModule) {
         })
     }
 
+    const main = options.mainModule[psModule.name];
+
+    if (main) {
+      js = `${js}\nmodule.exports.${main}(module)();`;
```

Comment:
Why does this call the function with `module`? Isn't `module.exports.${main}` an `Eff`?